client/src/components/scheduler/Overview.js

```diff
@@ -23,6 +23,18 @@ const Overview = ({ name, duration, description }) => {
         <AccessTime />
         <Typography variant="caption">{duration} minute</Typography>
       </Grid>
+      {appointmentTime ? (
+        <Grid
+          container
+          item
+          alignItems="center"
+          className={classes.iconWrapper}
+        >
+          <Typography variant="caption">
+            {JSON.stringify(appointmentTime)}
+          </Typography>
+        </Grid>
+      ) : null}
```

Comment:
a nicer way to do this is `{appointmentTime && (...)}` so we don't have to have the `: null`

Reply:
Will do!